readme.md

```diff
@@ -1067,6 +1079,46 @@ Also it will look for the same files under the kubernetes profile:
 - application-kubernetes.yml
 - application-kubernetes.yaml
 
+##### Vert.x & generic Java
+
+For generic java, if the @Dekorate annotion is present, then dekorate will look for confiugration uder:
+
+- application.properties
```

Comment:
Does it also work for `application.yaml`? (I suppose yes)

Reply:
Yes it does.